### DISCUSSION TITLE
Limitations os Solito

### DISCUSSION THREAD

**Discussion by rodolphoasb** [nandorojo/solito#112] [Q&A] [answered]
Hey @nandorojo, thank you for your awesome work on Solito!

I'm thinking about using it for a side project here in Brazil.

I would like to know what are the limitations of using Solito for the web, for example, can I use everything that I would use on a normal Nextj.js project (my current stack uses TailwindCSS and Next serverless with tRPC)? Can I upgrade to Next.js 12.2 (the starter project is still using 12.1)?

Thank you for your help!

**Comment by nandorojo** [comment, marked as answer]
push the limits :) you can do anything Next.js can

if you want it to work on native too, i suggest using nativewind as a tailwind solution (waiting on @marklawlor to finish his Tailwind + Solito PR example). i haven't tried upgrading to Next 12.2 yet, but I'm happy to review a PR / hear how it goes for you

**Reply by rodolphoasb** [reply]
Awesome! I'll give it a try on the weekend!

**Reply by nandorojo** [reply]
i added an example with tailwind & next 12.3!

it even has SSR support: https://github.com/nandorojo/solito/tree/master/example-monorepos/with-tailwind

**Reply by SunnyAureliusRichard** [reply]
hi @nandorojo . What's the difference between the project `blank` and `with-tailwind` when it comes to enabling SSR? I see that `blank` is lacking SSR and I've built my app on top of it.

**Reply by nandorojo** [reply]
depends. are you using dripsy for responsive design?

**Reply by SunnyAureliusRichard** [reply]
yes, but only in one place. I can do without it if that's the compromise I have to make

**Comment by Rychillie** [comment]
Hi @rodolphoasb , I'm Brazilian too, I've been using solito in some projects for a few months, I'm available for a conversation if you need it! I would be happy to see other Brazilians and companies here using this stack!

**Reply by raulgonzalezdev** [reply]
@Rychillie Hello, Friend, look I am Venezuelan, but I work remote for a company in your country CENTRAL IT, right now we just put into production an app made in NEXT-Js + Chakra and now we want to extend its cross-platform use to Android and IOS, could you share your experiences with Solito, we have not started yet but since I read about this project I have wanted to try the implementation, today I will create a local environment, to try to use solito. dev the web version of our app, if you could guide me it would be great, +584148398384 this is My Whatapps.

**Comment by verbiiz** [comment]
how did it go? were there any limitations of maintaining solito + nextjs? i'm also considering using solito at some point. i already switched away from react-native to nextjs to build quicker, but still want to revisit my mobile app.

**Reply by zakwear** [reply]
I'm also interested. What holds me back is RSC/SSR support. Not clear to me whether async components will be supported.
